src/React/Basic/DOM.purs

```diff
@@ -5,20 +5,48 @@
+import React.Basic.DOM.Internal (CSS, SharedProps, unsafeCreateDOMComponent) as Internal
 import Unsafe.Coerce (unsafeCoerce)
+import Web.DOM (Node)
+
+-- | Returns the current DOM node associated with the given
+-- | instance, or an Error if no node was found or the given
+-- | instance was not mounted.
+-- |
+-- | Note: Relies on `ReactDOM.findDOMNode`
+findDOMNode :: ComponentInstance -> Effect (Either Error Node)
+findDOMNode instance_ = try do
+  node <- runEffectFn1 findDOMNode_ instance_
+  case toMaybe node of
+    Nothing -> throw "Node not found."
```

Comment:
Since you're wrapping this in `try` anyway, you could use `note` here.

Reply:
I think it would need `note'` and probably more code, unless there's a trick for composing them I'm not seeing